integrationtests/as-lucene-directory/src/test/java/org/infinispan/test/integration/as/wildfly/InfinispanModuleMemberRegistrationIT.java

```diff
@@ -128,10 +128,52 @@ public void testNewMemberSearch() throws Exception {
    @Test
    @InSequence(value = 3)
    @OperateOnDeployment("dep.active-2")
+   public void testNewMemberLuceneSearch() throws Exception {
+      List<Member> search = memberRegistration.luceneSearch("Peter");
+
+      assertFalse("Expected at least one result after the indexing", search.isEmpty());
+      assertEquals("Lucene search hasn't found a member", "Peter O'Tall", search.get(0).getName());
+   }
+
+   @Test
+   @InSequence(value = 4)
+   @OperateOnDeployment("dep.active-2")
    public void testNonExistingMember() throws Exception {
       List<Member> search = memberRegistration.search("TotallyInventedName");
 
       assertNotNull("Search should never return null", search);
       assertTrue("Search results should be empty", search.isEmpty());
    }
+
+   @Test
+   @InSequence(value = 5)
+   @OperateOnDeployment("dep.active-2")
+   public void testLuceneNonExistingMember() throws Exception {
+      List<Member> search = memberRegistration.luceneSearch("TotallyInventedName");
+
+      assertNotNull("Search should never return null", search);
+      assertTrue("Search results should be empty", search.isEmpty());
+   }
+
+   @Test
+   @InSequence(value = 6)
+   @OperateOnDeployment("dep.active-2")
+   public void testPurgeIndex() throws Exception {
+      memberRegistration.purgeMemberIndex();
+      List<Member> search = memberRegistration.search("Peter");
+
+      assertNotNull("Search should never return null", search);
+      assertTrue("Search results should be empty", search.isEmpty());
+   }
+
+   @Test
+   @InSequence(value = 7)
+   @OperateOnDeployment("dep.active-2")
+   public void testReIndex() throws Exception {
```

Comment:
Hey @Sanne, I'm trying to figure out how to fix this now, but I don't understand how the testPurgeIndex() passes since it makes two invocations on memberRegistration as well?